Improve compatibility between pipeline & cluster (#21)
VIB-PSB/ksrates: README.md

           cp ksrates/test ~
           cd ~/test

> 2. Prepare the configuration files. The `test` directory already contains:

Reviewer: ```suggestion
2. Prepare the configuration files.

      The `test` directory already contains:
      
```

      * A pre-filled *ksrates* configuration file (`config_elaeis.txt`) for the oil palm use case.

>       * A Nextflow configuration file template (`nextflow.config`) to configure the executor to be used (i.e., a local computer or a compute cluster) and its resources made available to Nextflow such as the number of CPUs. It also configures whether to use the *ksrates* Singularity or Docker container. This may need to be adapted to your available resources.

Reviewer: ```suggestion
      * A Nextflow configuration file template (`nextflow.config`) to configure the executor to be used (i.e., a local computer or a compute cluster) and its resources made available to Nextflow such as the number of CPUs. It also configures whether to use the *ksrates* Singularity or Docker container. The configuration file may need to be adapted to your available resources.
```

       See the [full documentation](https://ksrates.readthedocs.io/) and the [Nextflow documentation](https://www.nextflow.io/docs/latest/config.html) for more detail on Nextflow configuration, e.g. for different HPC schedulers. We also provide additional, more general template Nextflow configuration files in the [doc](doc/source) directory in the repository.

3. Launch the *ksrates* Nextflow pipeline.

>    > **Note:** If this is the first time you launch the pipeline, Nextflow will first download *ksrates* in`$HOME/.nextflow` and the *ksrates* Singularity container in `work/singularity`.

Reviewer: ```suggestion
   > **Note:** If this is the first time you launch the pipeline, Nextflow will first download *ksrates* Nextflow pipeline and the *ksrates* Singularity or Docker container.
```

       
       nextflow run VIB-PSB/ksrates --config ./config_elaeis.txt
	   

>    The path to the *ksrates* configuration file is specified through the `--config` parameter, while as long as the Nextflow configuration file is named `nextflow.config` and located in the launching folder the file is automatically detected. Alternatively, the user can specify a custom file by using the `-C` option (see `Nextflow documentation <https://www.nextflow.io/docs/latest/cli.html#hard-configuration-override>`__).

Reviewer: ```suggestion
   The path to the *ksrates* configuration file is specified through the `--config` parameter. If the Nextflow configuration file is named `nextflow.config` and located in the launching folder the file is automatically detected. Alternatively, the user can specify a custom file by using the `-C` option (see `Nextflow documentation <https://www.nextflow.io/docs/latest/cli.html#hard-configuration-override>`__).
```

           cd ~/test

2. Launch the *ksrates* Nextflow pipeline.

   > **Note:** If this is the first time you launch the pipeline, Nextflow will first download *ksrates* and the *ksrates* Singularity container (in the directory specified by `NXF_SINGULARITY_CACHEDIR`, e.g. the launching directory).
       
       export NXF_SINGULARITY_CACHEDIR=$PWD
       nextflow run VIB-PSB/ksrates --config ./config_elaeis.txt -with-singularity vibpsb/ksrates:latest
	   
   The `test` directory already contains a pre-filled *ksrates* pipeline configuration file (`config_elaeis.txt`) for the oil palm use case, provided through the `--config` parameter.
   

>    The directory also contains a Nextflow configuration file (`nextflow.config`) to fine-tune executor resources and *ksrates* container settings. If needed, please adapt it according to the availbale resources; see the [full documentation](https://ksrates.readthedocs.io/) and the [Nextflow documentation](https://www.nextflow.io/docs/latest/config.html) for more detail on Nextflow configuration, e.g. for different HPC schedulers. The file is automatically detected; alternatively, the user can specify a custom filename or path through the `-c` option. By default, the pipeline is run on a local executor.

Reviewer: ```suggestion
   The `test` directory also already contains a Nextflow configuration file template(`nextflow.config`) to configure the executor to be used (i.e., a local computer or a compute cluster) and its resources made available to Nextflow such as the number of CPUs. It also configures whether to use the *ksrates* Singularity or Docker container. This may need to be adapted to your available resources.
   See the [full documentation](https://ksrates.readthedocs.io/) and the [Nextflow documentation](https://www.nextflow.io/docs/latest/config.html) for more detail on Nextflow configuration, e.g. for different HPC schedulers. We also provide additional, more general template Nextflow configuration files in the [doc](doc/source) directory in the repository.
   If the Nextflow configuration file is named `nextflow.config` the file is automatically detected. Alternatively, the user can specify a custom file by using the `-C` option (see `Nextflow documentation <https://www.nextflow.io/docs/latest/cli.html#hard-configuration-override>`__).
```

Reviewer: I think this would fit better in a "Prepare the config files section" as you had before.
